### PR TITLE
add `rohandebsarkar.github.io`

### PR DESCRIPTION
**Pull request by rohandebsarkar**
This PR is:

- [x] Adding a new domain
- [ ] Updating existing domain **size**
- [ ] Changing domain name
- [ ] Removing existing domain from list
- [ ] Website code changes (512kb.club site)
- [ ] Other not listed

- [ ] I used the uncompressed size of the site
- [ ] I have included a link to the GTMetrix report
- [ ] The domain is in the correct alphabetical order
- [ ] This site is not a ultra lightweight site
- [ ] The following information is filled identical to the data file

***I confirm that I have read the [FAQ section](https://512kb.club/faq), particularly the two red items around minimal pages and inappropriate content and I attest that this site is neither of these things.***

- [x] Check to confirm

```
- domain: rohandebsarkar.github.io
  url: https://rohandebsarkar.github.io/
  size: 98
  last_checked: 2022-10-28
```

GTMetrix Report: https://gtmetrix.com/reports/rohandebsarkar.github.io/B62Dlfcq/

### CODE REVIEW

**Comment by garritfra** [comment]
Please use the uncompressed size of the page, as described by our [FAQ](https://512kb.club/faq).

**Comment by garritfra** [comment]
It's right next to the compressed size.

File: _data/sites.yml [hunk 1]
```diff
@@ -2018,6 +2018,11 @@
   size: 243
   last_checked: 2022-04-26
 
+- domain: rohandebsarkar.github.io
+  url: https://rohandebsarkar.github.io/
+  size: 98
```
Comment (rohandebsarkar) [inline _data/sites.yml:2023]:
```suggestion
  size: 392
```

**Comment by rohandebsarkar** [comment]
Thank you for the reply. I have updated the size to reflect the uncompressed size of the page.

File: _data/sites.yml [hunk 2]
```diff
@@ -2018,6 +2018,11 @@
   size: 243
   last_checked: 2022-04-26
 
+- domain: rohandebsarkar.github.io
+  url: https://rohandebsarkar.github.io/
+  size: 392
```
Comment (rohandebsarkar) [inline _data/sites.yml:2023]:
```suggestion
  size: 226
```
I just updated my site. New GTmetrix report: https://gtmetrix.com/reports/rohandebsarkar.github.io/B62Dlfcq/

**Comment by garritfra** [comment]
Thanks!!

**Pull request merged**